packages/create-remix/__tests__/create-remix-test.ts

```diff
@@ -210,6 +210,25 @@ describe("create-remix CLI", () => {
     expect(fse.existsSync(path.join(notEmptyDir, "app/root.tsx"))).toBeFalsy();
   });
 
+  it("allows non-empty directories when --allow-non-empty is specified", async () => {
+    let notEmptyDir = getProjectDir("non-interactive-not-empty-dir-allowed");
+    fse.mkdirSync(notEmptyDir);
+    fse.createFileSync(path.join(notEmptyDir, "some-file.txt"));
+
+    let { status, stderr, stdout } = await execCreateRemix({
+      args: [notEmptyDir, "--no-install", "--no-git-init", "--allow-non-empty"],
+      interactive: false,
+    });
+
+    expect(stdout).toMatch(
```

Comment:
This should fix the failing Windows test, similar to other tests in this file.

```suggestion
    expect(
      stdout.trim().replace("<TEMP_DIR>\\", "<TEMP_DIR>/") // Normalize Windows path
    ).toMatch(
```